LOAN/LoanTypes/LoanProducts.rdf

```diff
@@ -103,30 +123,163 @@
+					</owl:Class>
+				</owl:someValuesFrom>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-loan-ln-ln;hasCost"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fnd-acc-cur;MonetaryAmount"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>line item</rdfs:label>
+		<skos:definition>description of a product or service including its unit cost, number of units and total cost</skos:definition>
+		<skos:editorialNote>Future consideration: move this to ProductsAndServices ontology (fibo-fnd-pas-pas).</skos:editorialNote>
+	</owl:Class>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanMarketCategory">
+		<rdfs:subClassOf rdf:resource="&fibo-fnd-arr-cls;IndustrySectorClassifier"/>
+		<rdfs:label>loan market category</rdfs:label>
+		<skos:definition>category indicating the market domain in which the loan product is offered</skos:definition>
+	</owl:Class>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Automobile">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>auto loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Commercial">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>commercial loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Construction">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>construction loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Education">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>education loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Mortgage">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>mortgage loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-Personal">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>personal loan</rdfs:label>
+	</owl:NamedIndividual>
+	
+	<owl:NamedIndividual rdf:about="&fibo-loan-typ-prod;LoanMarketCategory-SmallBusiness">
+		<rdf:type rdf:resource="&fibo-loan-typ-prod;LoanMarketCategory"/>
+		<rdfs:label>small business loan</rdfs:label>
+		<fibo-fnd-utl-av:explanatoryNote>This is related to the mortgage loan purpose: &apos;business or commercial&apos;.</fibo-fnd-utl-av:explanatoryNote>
+	</owl:NamedIndividual>
+	
+	<owl:Class rdf:about="&fibo-loan-typ-prod;LoanProduct">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-pas-fpas;FinancialProduct"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fbc-pas-fpas;isEmbodiedIn"/>
+				<owl:someValuesFrom rdf:resource="&fibo-loan-ln-ln;Loan"/>
```

Comment:
this does not make sense: you should be able to offer a loan product without having issued any actual loans. better in any case to link from the loan to the product that was sold.

Reply:
Agreed - I just moved this stuff out of its original home in LoanCore or in one of the other Loans ontologies - didn't look at it for whether or not it makes sense.  I think an SME would be needed to do that, but that wasn't the intent of this pull request.

Reply:
I think I've fixed this - changed the property to isRealizedBy, and the various restrictions to min 0